goto_handler.go

```diff
@@ -0,0 +1,116 @@
+package migrate
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+func (m *Migrate) HandleDirtyState(path string) {
+	// Perform actions when dirty is true
```

Comment:
Indentation looks off. You should run `go fmt` to properly reformat these files.